pkg/controller/windowsmachineconfig/windows/windows.go

```diff
@@ -84,7 +84,7 @@ func (vm *Windows) initializeBootstrapperFiles() error {
 
 	log.V(5).Info("stderr associated with the ignition file download", "stderr", stderr)
 	if len(stderr) > 0 {
-		log.Info(stderr, "err", "error while downloading the ignition file from cluster")
+		log.Info("error while downloading the ignition file from cluster", "err", stderr)
```

Comment:
Please make this `stderr` instead of `err`, so that it is clear we are looking at stderr output.